test_enclave/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "test_enclave"
+version = "0.2.0-pre0"
+edition = "2021"
+license = "Apache-2.0"
+
```

Comment:
All of the test enclave is more or less copy pasta from https://github.com/mobilecoinfoundation/sgx it seems that it may now be the time to prioritize the enclave building utilties.